test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveMapKey.cs

```diff
@@ -1738,5 +1741,14 @@ public struct ReceivedUpdates : IComponentData
                 get => Improbable.Gdk.Tests.ExhaustiveMapKey.ReferenceTypeProviders.UpdatesProvider.Get(handle);
             }
         }
+
+        internal class ExhaustiveMapKeyDynamic : IDynamicInvokable
+        {
+            public uint ComponentId => ExhaustiveMapKey.ComponentId;
+            public Action<Dynamic.IHandler> DynamicInvoker => (handler =>
+            {
+                handler.Accept(ExhaustiveMapKey.ComponentId, Serialization.Deserialize);
```

Comment:
If we expect users to deserialise `ComponentUpdateOp`s with this, giving them just the `Deserialize` function may not be correct. You will need to give both as you can have a partial `ComponentUpdate` but not a partial `ComponentData`

Reply:
Ah yep. Forgot to go back and add that in. Will do that now.

Reply:
Fixed now.